learn/deployment/aws-lambda.md

```diff
@@ -23,9 +23,34 @@ import ballerina/crypto;
 public function hash(awslambda:Context ctx, json input) returns json|error {
     return crypto:hashSha256(input.toJsonString().toBytes()).toBase16();
 }
+
+@awslambda:Function
+public function notifySQS(awslambda:Context ctx, awslambda:SQSEvent event) returns json {
+    return event.Records[0].body;
+}
+
+@awslambda:Function
+public function notifyS3(awslambda:Context ctx, awslambda:S3Event event) returns json {
+    return event.Records[0].s3.'object.key;
+}
+
+@awslambda:Function
+public function notifyDynamoDB(awslambda:Context ctx, awslambda:DynamoDBEvent event) returns json {
+    return event.Records[0].dynamodb.Keys.toString();
+}
+
+@awslambda:Function
+public function notifySES(awslambda:Context ctx, awslambda:SESEvent event) returns json {
+    return event.Records[0].ses.mail.commonHeaders.subject;
+}
+
+@awslambda:Function
+public function apigwRequest(awslambda:Context ctx, awslambda:APIGatewayProxyRequest request) {
+    io:println("Path: ", request.path);
+}
 ```
 
-The first parameter with the [awslambda:Context](/learn/api-docs/ballerina/awslambda/objects/Context.html) object contains the information and operations related to the current function execution in AWS Lambda such as the request ID and the remaining execution time. The second parameter with the `json` value contains the input request data. This input value format will vary depending on the source, which invoked the function e.g., an AWS S3 bucket update event. The return type of the function is `json|error`, which means in a successful scenario, the function can return a `json` value with the response or else in an error situation, the function will return an `error` value, which provides information on the error to the system.
+The first parameter with the [awslambda:Context](/learn/api-docs/ballerina/awslambda/objects/Context.html) object contains the information and operations related to the current function execution in AWS Lambda such as the request ID and the remaining execution time. The second parameter contains the input request data. This input value will vary depending on the source, which invoked the function (e.g., an AWS S3 bucket update event). The return type of the function is `json|error`, which means in a successful scenario, the function can return a `json` value with the response or else in an error situation, the function will return an `error` value, which provides information on the error to the system. You can also provide functions, which do not return anything at all, which implicitly signals a successful execution without a returning result.
```

Comment:
Better if we break this also to 2 paragraphs.

Reply:
+1, done.